contrib/integration-cli-on-swarm/agent/worker/worker.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/bfirsh/funker-go"
+	"github.com/docker/docker/contrib/integration-cli-on-swarm/agent/types"
+)
+
+func main() {
+	if err := xmain(); err != nil {
+		log.Fatalf("fatal error: %v", err)
+	}
+}
+
+func validImageDigest(s string) bool {
+	// FIXME
+	return strings.Contains(s, "sha256:")
```

Comment:
You could use `github.com/docker/distribution/reference` package here to validate the digest :angel: